README.md

```diff
@@ -26,6 +26,12 @@ A Loki-based logging stack consists of 3 components:
 Loki is like Prometheus, but for logs: we prefer a multidimensional label-based approach to indexing, and want a single-binary, easy to operate system with no dependencies.
 Loki differs from Prometheus by focussing on logs instead of metrics, and delivering logs via push, instead of pull.
 
+This stack is lightweight, can be used as a simple alternative to super-heavy and full-featured search stacks such as ELK or EFK:
```

Comment:
This is great stuff but I don't know if we want this on the front page. I defer to @tomwilkie on this one.

Reply:
wait for feedback ;)

Reply:
@Kuqd @tomwilkie any update?

Reply:
@cyriltovena  @slim-bean  @tomwilkie any update?